test/settings.py

```diff
@@ -5,6 +5,7 @@
 L1_HOST="localhost"
 L1_PORT="8545"
 
-GATEWAY_URL=f"http://{HOST}:{PORT}"
-FEEDER_GATEWAY_URL=f"http://{HOST}:{PORT}"
+APP_URL=f"http://{HOST}:{PORT}"
+GATEWAY_URL=APP_URL
+FEEDER_GATEWAY_URL=APP_URL
```

Comment:
Not sure why there are no spaces here around the equal symbols, they should be introduced.

Reply:
[935d87a](https://github.com/Shard-Labs/starknet-devnet/pull/64/commits/935d87a2bd9c20a8b210833d8ae804f444ab82d8)